service/application/src/main/java/com/solace/maas/ep/event/management/agent/command/SempDeleteCommandManager.java

```diff
@@ -287,7 +291,30 @@ private void executeDeleteRdpRestConsumer(Command command, SempApiProvider sempA
         try {
             restDeliveryPointApi.deleteMsgVpnRestDeliveryPointRestConsumer(request.getMsgVpn(), request.getRdpName(), request.getRestConsumerName());
         } catch (ApiException e) {
-            handleSempApiDeleteException(e, "Rest Consumer", request.getRdpName());
+            handleSempApiDeleteException(e, "Rest Consumer");
+        }
+    }
+
+    private void executeDeleteRdpOauthJwtClaims(Command command, SempApiProvider sempApiProvider) throws ApiException, JsonProcessingException {
+        RestDeliveryPointApi restDeliveryPointApi = sempApiProvider.getRestDeliveryPointApi();
+        SempRdpOauthJwtClaimDeletionRequest request = objectMapper.readValue(
+                objectMapper.writeValueAsString(command.getParameters().get(SEMP_DELETE_DATA)),
+                SempRdpOauthJwtClaimDeletionRequest.class);
+
+        Validate.notEmpty(request.getMsgVpn(), "Msg VPN must not be empty");
```

Comment:
Please address [Sonar recommendations](https://sonarq.solace.com/project/issues?id=SolaceProducts_event-management-agent&pullRequest=247&resolved=false) (one of them was introducing a constant for this, and there are some other good suggestions there as well!)